src/taipy/core/_version/_version_cli.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2022 Avaiga Private Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+# the License. You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+import uuid
+
+import click
+
+from taipy.logger._taipy_logger import _TaipyLogger
+
+from ..cycle._cycle_manager_factory import _CycleManagerFactory
+from ..data._data_manager_factory import _DataManagerFactory
+from ..job._job_manager_factory import _JobManagerFactory
+from ..pipeline._pipeline_manager_factory import _PipelineManagerFactory
+from ..scenario._scenario_manager_factory import _ScenarioManagerFactory
+from ..task._task_manager_factory import _TaskManagerFactory
+from ._version_manager import _VersionManager
+
+
+def skip_run(ctx, param, value):
+    if not value or ctx.resilient_parsing:
+        return
+
+    # TODO: Copy the current latest version to the new version
+    ctx.exit()
+
+
+def clean_all_entities_by_version(version_number):
+    """Delete all entities belongs to a version from the Taipy data folder."""
+    _JobManagerFactory._build_manager()._delete_by_version(version_number)
+    _ScenarioManagerFactory._build_manager()._delete_by_version(version_number)
+    _PipelineManagerFactory._build_manager()._delete_by_version(version_number)
+    _TaskManagerFactory._build_manager()._delete_by_version(version_number)
+    _DataManagerFactory._build_manager()._delete_by_version(version_number)
+
+
+@click.command()
+@click.option(
+    "--development",
+    "-d",
+    "mode",
+    flag_value="development",
+    default=True,
+    help="Execute Taipy application in DEVELOPMENT mode. This is the default behavior.",
+)
+@click.option(
+    "--experiment", "-e", "mode", flag_value="experiment", help="Execute Taipy application in EXPERIMENTAL mode."
```

Comment:
Same comment for experimental mode. I know it will be eventually on the official doc, but I think it would be nice for the user to have it hand on the cli.